registration/templates/registration/registration_form.html

```diff
@@ -20,6 +20,9 @@
         {% include row with field=form.accept_terms only %}
         {% include row with field=form.subscribe_to_updates only %}
     {% endwith %}
+    {% with row="registration/hidden_field.html" %}
```

Comment:
Do we need a separate template just for a hidden form field? Why not just include it directly?